detectable/src/test/java/com/synopsys/integration/detectable/detectables/npm/lockfile/functional/NpmOutputParserTest.java

```diff
@@ -24,23 +24,65 @@
 import com.synopsys.integration.detectable.detectables.npm.cli.parse.NpmCliParser;
 import com.synopsys.integration.detectable.detectables.npm.lockfile.model.NpmParseResult;
-import com.synopsys.integration.detectable.util.FunctionalTestFiles;
-import com.synopsys.integration.detectable.util.GraphCompare;
+import com.synopsys.integration.detectable.util.graph.NameVersionGraphAssert;
 
 public class NpmOutputParserTest {
     @Test
-    @Disabled
     public void npmCliDependencyFinder() {
         final NpmCliParser parser = new NpmCliParser(new ExternalIdFactory());
-        final String testIn = FunctionalTestFiles.asString("/npm/packman_proj_dependencies.json");
+        final String testIn =
+            "{" + "\n" +
+                "   \"name\": \"node-js\"," + "\n" +
+                "   \"version\": \"0.2.0\"," + "\n" +
+                "   \"dependencies\": {" + "\n" +
+                "       \"upper-case\": {" + "\n" +
+                "       \"version\": \"1.1.3\"," + "\n" +
+                "       \"from\": \"upper-case@latest\"," + "\n" +
+                "       \"resolved\": \"https://registry.npmjs.org/upper-case/-/upper-case-1.1.3.tgz\"" + "\n" +
+                "       }," + "\n" +
+                "       \"xml2js\": {" + "\n" +
+                "           \"version\": \"0.4.17\"," + "\n" +
+                "           \"from\": \"xml2js@latest\"," + "\n" +
+                "           \"resolved\": \"https://registry.npmjs.org/xml2js/-/xml2js-0.4.17.tgz\"," + "\n" +
+                "            \"dependencies\": {" + "\n" +
+                "               \"sax\": {" + "\n" +
+                "                   \"version\": \"1.2.2\"," + "\n" +
+                "                   \"from\": \"sax@>=0.6.0\"," + "\n" +
+                "                   \"resolved\": \"https://registry.npmjs.org/sax/-/sax-1.2.2.tgz\"" + "\n" +
+                "               }," + "\n" +
+                "               \"xmlbuilder\": {" + "\n" +
+                "                   \"version\": \"4.2.1\"," + "\n" +
+                "                   \"from\": \"xmlbuilder@>=4.1.0 <5.0.0\"," + "\n" +
+                "                   \"resolved\": \"https://registry.npmjs.org/xmlbuilder/-/xmlbuilder-4.2.1.tgz\"," + "\n" +
+                "                   \"dependencies\": {" + "\n" +
+                "                       \"lodash\": {" + "\n" +
+                "                           \"version\": \"4.17.4\"," + "\n" +
+                "                           \"from\": \"lodash@>=4.0.0 <5.0.0\"," + "\n" +
+                "                           \"resolved\": \"https://registry.npmjs.org/lodash/-/lodash-4.17.4.tgz\"" + "\n" +
+                "                       }" + "\n" +
+                "                   }" + "\n" +
+                "               }" + "\n" +
+                "           }" + "\n" +
+                "       }" + "\n" +
+                "   }" + "\n" +
+                "}";
```

Comment:
I'd prefer something like the following.
```
StringUtils.join(
            "line1",
            "line2",
            "line3"
        , System.lineSeparator()
);
```
That's a lot of String concatenation, its hard to read, and the "\n" might cause issues on Windows.